test/test_torchscript_consistency.py

```diff
@@ -99,7 +99,8 @@ def func(_):
             f_max = 20.0
             n_mels = 10
             sample_rate = 16000
-            return F.create_fb_matrix(n_stft, f_min, f_max, n_mels, sample_rate)
+            norm = None
```

Comment:
If you're exercising torchscript I'd pass the less trivial type which is a string instead of None.

Reply:
You mean: default to empty string? We don't use that elsewhere, but sounds good to me.

Reply:
done